app/app_test.go

```diff
@@ -48,3 +48,26 @@ func TestFyneApp_SetIcon(t *testing.T) {
 
 	assert.Equal(t, setIcon, app.Icon())
 }
+
+func TestFynaApp_Clipboard(t *testing.T) {
+	app := NewWithID("io.fyne.test")
+	app.NewWindow("test")
```

Comment:
I haven't tried but I suspect you might want to use `test.NewTempApp` and `test.NewTempWindow`?

Reply:
Both actually work, but I have updated the test with your suggestion.

Reply:
Both will work, but the ones Jacob suggested clean up at the end of the test for you :)